orchagent/main.cpp

```diff
@@ -140,6 +145,121 @@ void init_gearbox_phys(DBConnector *applDb)
     delete tmpGearboxTable;
 }
 
+bool getSystemPortConfigList(vector<sai_system_port_config_t> &sysportcfglist)
+{
+    DBConnector cfgDb("CONFIG_DB", 0);
+    DBConnector appDb("APPL_DB", 0);
+
+    Table cfgDeviceMetaDataTable(&cfgDb, CFG_DEVICE_METADATA_TABLE_NAME);
+    Table cfgSystemPortTable(&cfgDb, CFG_SYSTEM_PORT_TABLE_NAME);
+    Table appSystemPortTable(&appDb, APP_SYSTEM_PORT_TABLE_NAME);
+
+    //At this point of time (i.e, at the time of orchagent start), the CONFIG_DB is completely fully
+    //populated with the contents of config_db.json.
+
+    string value;
+    //Get the swith type.
+    if(!cfgDeviceMetaDataTable.hget("localhost", "switch_type", value))
```

Comment:
It seems not have consistent format with Sonic/other changes in the same diff --> space between "if" and "(!cfgDeviceMetaDataTable...)

if (!cfgDeviceMetaDataTable.hget("localhost", "switch_type", value))

Reply:
Yes. We'll fix this

Reply:
Fixed